src/Database/Relations/BelongsToMany.php

```diff
@@ -100,6 +100,110 @@ public function create(array $attributes = [], array $pivotData = [], $sessionKe
         return $model;
     }
 
+    /**
+     * @param mixed $id
+     * @param array $attributes
+     * @param bool  $touch
+     */
+    public function attach($id, array $attributes = [], $touch = true)
+    {
+        $insertData = $this->formatAttachRecords($this->parseIds($id), $attributes);
+        $elementIDList = array_pluck($insertData, $this->relatedPivotKey);
```

Comment:
Could you give me more information on what exactly this variable holds? `$elementIDList` isn't very clear to me what's going on.

Reply:
I receive IDs of elements that have been attached.
For example:
```
$obProduct->attach(1); //$elementIDList == [1];
$obProduct->attach([1,3]); //$elementIDList == [1,3];
$obProduct->attach([1 => ['pivot_field' => 'test'], 2, 5, 10]); //$elementIDList == [1, 2, 5,10];
```

Reply:
Please change to `$attachedIDs` instead then

Reply:
@kharanenka not $attachedIDList, $attachedIDs. October does not include the type of variable in the name of the variable anywhere in it's code base so I don't want to start doing it here.